### DISCUSSION TITLE
Why I can't `jump to definition (F12)`?

### DISCUSSION THREAD

**Discussion by GF-Huang** [dahall/Vanara#362] [Q&A] [answered]
As you see below, I want to jump to definition of `WinHttpSetStatusCallback` to see the `WINHTTP_STATUS_CALLBACK` details, but I jump into the partial implementation. Is there any workaround?

![devenv_VHoOXjpEFs](https://user-images.githubusercontent.com/4510984/210158354-50c9f57e-c0d4-4ae4-bafc-19f6a8b179dc.gif)

![image](https://user-images.githubusercontent.com/4510984/210158373-c20b5598-800a-42ad-b24a-e9ff3aff2f8b.png)

**Comment by dahall** [comment]
Hm. Don't know. It works in my instance of the same version of Visual Studio 2022.

What app did you use to do the animated screen capture? That is very useful.

**Reply by GF-Huang** [reply]
> Hm. Don't know. It works in my instance of the same version of Visual Studio 2022.

Do you try with a new project that install Vanara by NuGet? Rather than reference the local source code of Vanara.

> What app did you use to do the animated screen capture? That is very useful.

Both [Capture](https://mathewsachin.github.io/Captura/) and [ShareX](https://getsharex.com/), they're good tools.

**Reply by dahall** [reply]
> Do you try with a new project that install Vanara by NuGet? Rather than reference the local source code of Vanara.

Yes. Both ways work in my environment.
> Both [Capture](https://mathewsachin.github.io/Captura/) and [ShareX](https://getsharex.com/), they're good tools.

Thanks!!

**Reply by GF-Huang** [reply]
> Yes. Both ways work in my environment.

Oh, thanks, I will check further on my environment.

**Comment by dahall** [comment, marked as answer]
Using your tool recommendation, I captured what I see with a NuGet provisioned project:
![devenv_TAkxoizl0p](https://user-images.githubusercontent.com/14349943/212502805-0da72260-2c73-4888-9ad3-bac1ae8eb84c.gif)

**Reply by GF-Huang** [reply]
I found the difference, mine is `embedded`, yours are `decompiled`. But I don't know what the details differences.

![image](https://user-images.githubusercontent.com/4510984/212517918-c0198168-7e00-493a-b52b-6ce71f0a8efc.png)
![image](https://user-images.githubusercontent.com/4510984/212518001-3d232e01-ef8d-4b39-a67a-687fd6e65970.png)

**Reply by GF-Huang** [reply]
God, I found why. Just **uncheck** the red box option as below, all works well.

![image](https://user-images.githubusercontent.com/4510984/212519691-b8e0771e-49de-451e-8f65-4b79cadada19.png)

**Reply by dahall** [reply]
Great hint. Thanks for posting.
